docs/writing-stories/index.mdx

```diff
@@ -204,13 +204,13 @@ Addons can enhance args. For instance, [Actions](../essentials/actions.mdx) auto
 
 ### Using the play function
 
-Storybook's `play` function and the [`@storybook/addon-interactions`](https://storybook.js.org/addons/@storybook/addon-interactions) are convenient helper methods to test component scenarios that otherwise require user intervention. They're small code snippets that execute once your story renders. For example, suppose you wanted to validate a form component, you could write the following story using the `play` function to check how the component responds when filling in the inputs with information:
+Storybook's `play` function and the [Component tests panel](../writing-tests/component-testing.mdx) are convenient helper methods to test component scenarios that otherwise require user intervention. They're small code snippets that execute once your story renders. For example, suppose you wanted to validate a form component, you could write the following story using the `play` function to check how the component responds when filling in the inputs with information:
```

Comment:
This is just a minor item, and feel free to disregard it, but I wonder if the component tests panel link shouldn't be linked to the interactive debugger section in the component tests documentation for accuracy.